### PR TITLE
Add 'install' make target

### PR DESCRIPTION
**Pull request by drons**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -135,3 +135,19 @@ if (USE_TESTS)
   add_test(NAME gui_test_general COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/tests/run_gui_tests.py" $<TARGET_FILE:qtmonkey_app> $<TARGET_FILE:test_app> "${CMAKE_CURRENT_SOURCE_DIR}/tests/test1.js")
   add_test(NAME gui_test_restart COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/tests/gui_restart_test.py" $<TARGET_FILE:qtmonkey_app> $<TARGET_FILE:test_app> "${CMAKE_CURRENT_SOURCE_DIR}/tests/test_restart.js")
 endif ()
+
+file(GLOB QT_MONKEY_HEADERS ${qt_monkey_SOURCE_DIR}/*.hpp)
+install(FILES ${QT_MONKEY_HEADERS} DESTINATION include/qt_monkey)
+
+install(FILES ${qt_monkey_SOURCE_DIR}/contrib/json11/json11.hpp DESTINATION include/json11)
```
Comment (Dushistov) [inline CMakeLists.txt]: This is private header, json11 library internally used for cooperation between qtmonkey application and agent,
so no sense to export it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -135,3 +135,19 @@ if (USE_TESTS)
...
+
+install(FILES ${qt_monkey_BINARY_DIR}/libcommon_app_lib.a
```
Comment (Dushistov) [inline CMakeLists.txt]: libcommon_app_lib.a is used to share code between qtmonkey_app and qtmonkey_gui,
so there is no sense to export/install it.

**Review by Dushistov — CHANGES_REQUESTED** [review]
Thank you for contribution, there are some issues that I would like to resolve before merging PR.

**Comment by drons** [comment]
Thanks!

**Pull request merged**